build/stf-run-ci/tasks/main.yml

```diff
@@ -159,7 +138,7 @@
           pod-security.kubernetes.io/audit: restricted
           pod-security.kubernetes.io/warn: restricted
 
-- when: __deploy_from_index_enabled | bool
+- when: __deploy_from_index_enabled | bool and __local_build_enabled | bool
```

Comment:
This looks like the behaviour for __local_build_enabled is being changed.

This is needed because `__deploy_from_index_enabled==true` previous implied `__local_build_enabled==true` previously.